docs/docusaurus/docs/contributing/application_integration_support.md

```diff
@@ -0,0 +1,120 @@
+
+| GX Cloud                    | GX OSS                          |
+|-----------------------------|---------------------------------|
+| N/A                         | 3.8 to 3.11                     | 
+
+### GX versions
+
+The following table defines the GX versions supported by GX Cloud and GX OSS.
+
+| GX Cloud                    | GX OSS                          |
+|-----------------------------|---------------------------------|
+| N/A                         | - 0.17<br/> - 0.18              | 
+
+### Integrations
+
+The following table defines the supported GX Cloud and GX OSS integrations.
+
+
+| Application Type                         | GX Cloud             | GX OSS          |
+|------------------------------------------|----------------------|-----------------|
+| Data Source                              | Snowflake           | - Snowflake<br/> - Generic SQL ¹               |
+| Configuration Stores                     | N/A                 | - Filesystem<br/> - AWS S3GCS Buckets<br/> - Azure Blob Store                |
+| Communication (notifications)            | N/A ²               | - Slack (local) ³<br/> - email (local)<br/> - Microsoft Teams (local)<br/> - PagerDuty (local)                 |
+| Orchestrator                              | Airflow ⁴           | Airflow ⁴       |
+
+
+¹ Connection strings for various SQL Data Sources are supported, but dialect-specific SQL commands are not.<br/>
+² Support for Slack, email, and Zapier are planned for future releases.<br/>
+³ When used with the GX OSS library.<br/>
+⁴ Although only Airflow is supported, GX Cloud and GX OSS should work with any orchestrator that executes Python code.
+
+### GX components
+
+The following table defines the GX components supported by GX Cloud and GX OSS.
+
+| Component                                | GX Cloud             | GX OSS          |
+|------------------------------------------|----------------------|-----------------|
+| Expectations                             | See [Available Expectations](/docs/cloud/expectations/manage_expectations#available-expectations). | See [Create Expectations](/docs/guides/expectations/expectations_lp).                |
+| GX Agent                                 | All versions               | N/A        |
+
+
+## Community supported
+
+The following integrated applications, operating systems, and programming languages are supported by the community.
+
+### Operating systems
+
+The following table defines the operating systems supported by the community.
+
+| GX Cloud                       | GX OSS                        |
+|--------------------------------|-------------------------------|
+| Windows ¹                      | Windows ¹                     | 
+
+¹ Untested and unsupported by GX.
+
+### Integrations
+
+The following table defines the GX Cloud and GX OSS integrations supported by the community.
+
+
+| Application Type                         | GX Cloud             | GX OSS          |
```

Comment:
All GX Cloud is N/A.

Explain why N/A. All Community-supported integrations are applicable only to GX OSS.